ocs_ci/deployment/hosted_cluster.py

```diff
@@ -778,15 +778,16 @@ def setup_storage_client(self):
 
         # starting from ODF 4.16 on StorageClient creation Storage Claims created automatically
```

Comment:
I think it would be nice to update this comment with the information on 4.19

Reply:
done